docs/docs-content/getting-started/aws/aws.md

```diff
@@ -22,6 +22,8 @@ introducing you with Palette workflows and capabilities.
 In this section, you learn how to create a cluster profile. Then, you deploy a cluster to AWS by using Palette. Once
 your cluster is deployed, you can update it using cluster profile updates.
 
+<!-- vale off -->
```

Comment:
This never occurred to me LMAO

Reply:
:( I think that rule was introduced recently